packages/components/src/components/form/CleanSelect/index.tsx

```diff
@@ -109,6 +109,7 @@ interface Props extends Omit<SelectProps, 'components'> {
     wrapperProps?: Record<string, any>;
     variant?: InputVariant;
     minWidth?: string;
+    maxSearchLength?: number;
```

Comment:
this change is already in develop

Reply:
You are right. I don't know why it shows it as a difference when it is exactly the same as in develop. And the same applies to the comment above.

Reply:
i think rebase may help :)

Reply:
Resolved by new merge with develop